upup/pkg/fi/cloudup/networking.go

```diff
@@ -160,19 +164,23 @@ func findCNIAssets(c *api.Cluster, assetBuilder *assets.AssetBuilder) (*url.URL,
 	}
 
 	var cniAsset, cniAssetHash string
-	if util.IsKubernetesGTE("1.15", *sv) {
+	if util.IsKubernetesGTE("1.18", *sv) {
+		cniAsset = defaultCNIAssetK8s1_18
+		cniAssetHash = defaultCNIAssetSHA256StringK8s1_18
+		klog.V(2).Infof("Adding default CNI asset for k8s >= 1.18: %s", cniAsset)
+	} else if util.IsKubernetesGTE("1.15", *sv) {
 		// We're still on the same asset, but we use sha256
 		cniAsset = defaultCNIAssetK8s1_11
 		cniAssetHash = defaultCNIAssetSHA256StringK8s1_11
-		klog.V(2).Infof("Adding default CNI asset for k8s >= 1.11: %s", defaultCNIAssetK8s1_9)
+		klog.V(2).Infof("Adding default CNI asset for 1.18 > k8s >= 1.11: %s", cniAsset)
```

Comment:
```suggestion
		klog.V(2).Infof("Adding default CNI asset for 1.18 > k8s >= 1.15: %s", cniAsset)
```

Reply:
It is actually the same version, just the hash changed from sha1 to sha256, right?

Reply:
ah you're right, nevermind